index.html

```diff
@@ -6903,6 +6912,12 @@ <h2>Change log</h2>
       <a data-cite="JSON-LD11#embedding-json-ld-in-html-documents">Embedding JSON-LD in HTML Documents</a> [[JSON-LD11]]
       for treating script elements as a single document,
       as described in <a href="#change_4">Candidate Correction 4</a>.</li>
+    <li>2025-01-25: Correct some corner cases in transforming RDF Number Literals
+      to JSON numbers when <a data-link-for="JsonLdOptions">useNativeTypes</a> is `true`,
+      as described in <a href="#change_5">Candidate Correction 5</a>.</li>
```

Comment:
```suggestion
```
I suspect this has leaked from another PR, right?
It should probably be removed from that one.

Reply:
This was added in 3b0a95c which I didn't make. Do you want me to fix it?

Reply:
I added it to simplify a future merge, otherwise, it will lead to a conflict. If we do the PRs in an appropriate order, these will mesh together.